.gitignore

```diff
@@ -7,3 +7,5 @@ database/data
 # These should only exist in the frontend folder
 package.json
 package-lock.json
+
+data/
```

Comment:
Had it ignore the database file, but would like to make sure updating the .gitignore was appropriate in this instance.

Reply:
You should remove this after you update your branch.